src/bin/annotate.pl

```diff
@@ -149,13 +147,13 @@
 else{
 	`cat $out_dir/*/unfiltered.fusion.txt > $unfiltered_file` unless(-s $unfiltered_file);
 }
+print "UFF: $unfiltered_file\n"; 
```

Comment:
Maybe change `UFF` here as it does not seem evident that this is the an abbreviation for the unfiltered file without seeing the code itself.